src/target/riscv/debug_register_printers.h

```diff
@@ -0,0 +1,8 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include <stdbool.h>
+
+#include "debug_defines.h"
+
+unsigned int riscv_debug_reg_to_s(char *buf, enum riscv_debug_reg_ordinal reg_ordinal,
+		riscv_debug_reg_ctx_t context, uint64_t value);
```

Comment:
Since this is the programmer-facing function, can you add a comment that explains how to call this function, with a tiny example as well?

Reply:
Addressed.